RubeesEat.IntegrationTests/PageObjects/BillDetailsPageObject.cs

```diff
@@ -0,0 +1,33 @@
+using OpenQA.Selenium;
+using RubeesEat.IntegrationTests.WebTesting;
+using RubeesEat.Model;
+
+namespace RubeesEat.IntegrationTests.PageObjects;
+
+[TestFixture]
+public class BillDetailsPageObject : PageObject
+{
+    public string GetPersonWhoPayed()
+    {
+        return FindElement(By.CssSelector(".personWhoPayed")).Text;
+    }
+    public string GetDescription()
+    {
+        return FindElement(By.CssSelector(".description")).Text;
+    }
+
+    public string GetDate()
+    {
+        return FindElement(By.CssSelector(".date")).Text;
+    }
+    public string GetAmount()
+    {
+        return FindElement(By.CssSelector(".amount")).Text;
+    }
```

Comment:
For "values" that are just read from the page, I would suggest using properties instead of methods.